apps/opik-frontend/src/hooks/useLLMProviderModelsData.ts

```diff
@@ -1492,17 +1491,17 @@ export const PROVIDER_MODELS: PROVIDER_MODELS_TYPE = {
     },
   ],
 
-  [PROVIDER_TYPE.OLLAMA]: [
-    // the list will be full filled base on data in localstorage
+  [PROVIDER_TYPE.CUSTOM]: [
+    // the list will be full filled base on provider config response
   ],
 };
 
 const useLLMProviderModelsData = () => {
-  const { localModels, getLocalAIProviderData } = useLocalAIProviderData();
+  const customProviderModels = useCustomProviderModels();
 
   const getProviderModels = useCallback(() => {
```

Comment:
do we need it to be a usecallback? previously, it made sense as we called it from a localhost and needed to get a changed version all the time. however, now, if we receive it from BE, it would make sense to have it as `useMemo`